makepkg/PKGBUILD

```diff
@@ -15,7 +15,6 @@ depends=('git' 'pacutils' 'curl' 'bash' 'perl' 'perl-json-xs')
 makedepends=('git')
 optdepends=('bash-completion: bash completion'
             'zsh: zsh completion'
-            'devtools: aur-chroot'
```

Comment:
I'd only remove this when `aur-chroot--standalone` becomes the default implementation